nodeJS/introduction_to_nodeJS/debugging_node.md

```diff
@@ -12,9 +13,10 @@ This section contains a general overview of topics that you will learn in this l
 
 <div class="lesson-content__panel" markdown="1">
 
-- Watch [this video tutorial about Node.js debugging in VS Code](https://www.youtube.com/watch?v=2oFKNL7vYV8&ab_channel=VisualStudioCode) to see the process in action.
+- Watch [video tutorial about Node.js debugging in VS Code](https://www.youtube.com/watch?v=2oFKNL7vYV8&ab_channel=VisualStudioCode) to see the process in action.
 - Check out the official [VS Code Node debugger documentation](https://code.visualstudio.com/docs/nodejs/nodejs-debugging). Take note of the Javascript Debug Terminal- this is an easy way to get the debugger running!
-- Read [this article](https://web.archive.org/web/20230923074524/https://www.section.io/engineering-education/debug-node-devtools/) for a tutorial on how to debug Node with Chrome. NOTE: You can ignore anything that refers to debugging remote apps- you'll be debugging local apps.
+Read this article for a tutorial on [how to debug Node with Chrome](https://web.archive.org/web/20230923074524/https://www.section.io/engineering-education/debug-node-devtools/).
+NOTE: You can ignore anything that refers to debugging remote apps- you'll be debugging local apps.
```

Comment:
```suggestion
- Read this article for a tutorial on [how to debug Node with Chrome](https://web.archive.org/web/20230923074524/https://www.section.io/engineering-education/debug-node-devtools/). NOTE: You can ignore anything that refers to debugging remote apps- you'll be debugging local apps.

```
Need to keep these on the same line as well as make sure we still have the `-` to mark it as an item in a ul. Also popping a blank line after it since it'll fix one linting error - might as well while we're at it.